source/_integrations/coinbase.markdown

```diff
@@ -9,46 +9,35 @@ ha_iot_class: Cloud Polling
 
 ## Configuration
 
-To set it up, add the following information to your `configuration.yaml` file:
+The Coinbase integration is configured through Home Assistant's user interface, go the **Configuration** page then the **Integrations** page, click the `+` button and select Coinbase from the list of integrations. There will be a short delay while the integration's dependencies are installed after which you will be presented with a form to enter the required information to configure the integration.
 
-```yaml
-# Example configuration.yaml entry
-coinbase:
-  api_key: YOUR_API_KEY
-  api_secret: YOUR_API_SECRET
-```
+### Configuration Values
 
-{% configuration %}
-api_key:
-  description: Your API key to access coinbase.
-  required: true
-  type: string
-api_secret:
-  description: Your API secret to access coinbase.
-  required: true
-  type: string
-account_balance_currencies:
-  description: List of currencies to create account wallet sensors for.
-  required: false
-  type: list
-  default: all account wallets
-exchange_rate_currencies:
-  description: List of currencies to create exchange rate sensors for.
+{% configuration_basic %}
+API Key:
+  description: Your API key generated by Coinbase, this is required.
+API Secret:
+  description: Your API secret generated by Coinbase, this is required.
+Account Balance Currencies:
+  description: List of currency codes to create account wallet sensors for. This is optional, if left blank the integration will create sensors for all currencies you configured API key for. Separate multiple currencies with a comma `,` e.g., `BTC, EUR, ATOM`, codes should match those listed in the API key popup for cryptocurrencies or [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) for physical currencies.
+Exchange Rates:
+  description: Optional list of currencies to create exchange rate sensors for. Separate multiple currencies with a comma `,` e.g., `BTC, EUR, ATOM`, codes should match those listed in the API key popup for cryptocurrencies or [ISO 4217](https://en.wikipedia.org/wiki/ISO_4217) for physical currencies.
   required: false
   type: list
-{% endconfiguration %}
+{% endconfiguration_basic %}
+
 
-Possible currencies are codes that conform to the ISO 4217 standard where possible. Currencies which have or had no representation in ISO 4217 may use a custom code (e.g.,  BTC). A list of values can be obtained via https://api.coinbase.com/v2/currencies, for more information visit [the Coinbase API documentation](https://developers.coinbase.com/api/v2#get-currencies).
+### Text-Based Configuration (Deprecated)
```

Comment:
Just remove this section, we don't keep changelogs and such in the general documentation.